app/lib/features/chat/widgets/text_message_builder.dart

```diff
@@ -131,43 +130,48 @@ class _TextWidget extends ConsumerWidget {
     final emojiTextStyle = client.userId().toString() == message.author.id
         ? Theme.of(context).chatTheme.sentEmojiMessageTextStyle
         : Theme.of(context).chatTheme.receivedEmojiMessageTextStyle;
+
     return Column(
       children: [
-        ConstrainedBox(
-          constraints: BoxConstraints(maxWidth: messageWidth.toDouble()),
-          child: enlargeEmoji
-              ? Text(
-                  message.text,
-                  style: emojiTextStyle.copyWith(
-                    overflow: isReply ? TextOverflow.ellipsis : null,
-                    fontFamily: emojiFont,
-                  ),
-                  maxLines: isReply ? 3 : null,
-                )
-              : Html(
-                  onLinkTap: (url) => onLinkTap(url, context, ref),
-                  backgroundColor: Colors.transparent,
-                  data: message.text,
-                  pillBuilder: ({
-                    required String identifier,
-                    required String url,
-                    OnPillTap? onTap,
-                  }) =>
-                      _pillBuilder(context, ref, identifier, url),
-                  shrinkToFit: true,
-                  defaultTextStyle:
-                      Theme.of(context).textTheme.bodySmall!.copyWith(
-                            overflow: isReply ? TextOverflow.ellipsis : null,
-                            color: isNotice
-                                ? Theme.of(context)
-                                    .colorScheme
-                                    .onSurface
-                                    .withOpacity(0.5)
-                                : null,
-                          ),
-                  maxLines: isReply ? 3 : null,
+        enlargeEmoji
+            ? Text(
```

Comment:
so... you removed `messageWidth` being used, but it is still part of the widget's _required_ parameters ... why?

and doesn't this means messages now occupy the entire width? I don't think we want that..

Reply:
Oh I think this was unintentional. Apologies! I will undo it.